=

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
```

Comment:
just noticed this isnt the navigation drawer component, heed my comments but delete this file.

Reply:
I'm not sure what you mean, I'm supposed to delete what file? the NavigationDrawer?

Reply:
in the project, there is a file called `=` in the root of the project. delete this file

Reply:
oh lol I don't know how that happened. will do